### PR TITLE
dev-python/typing: version bump 3.7.4.1

### PR DESCRIPTION
**Pull request by TheChymera**
Package-Manager: Portage-2.3.76, Repoman-2.3.17
Signed-off-by: Horea Christian <chr@chymera.eu>

### CODE REVIEW

**Comment by TheChymera** [comment]
@mgorny @SoapGentoo  can you help me get this merged?

File: dev-python/typing/typing-3.7.4.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (mgorny) [inline dev-python/typing/typing-3.7.4.1.ebuild]: Why not 7?

File: dev-python/typing/typing-3.7.4.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+PYTHON_COMPAT=( python2_7 python3_6 pypy )
+
+inherit distutils-r1
+
+DESCRIPTION="Type Hints for Python"
```
Comment (mgorny) [inline dev-python/typing/typing-3.7.4.1.ebuild]: I presume this includes backports from 3.7, correct?

File: dev-python/typing/typing-3.7.4.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+LICENSE="PSF-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86 ~amd64-fbsd ~amd64-linux ~x86-linux ~x64-solaris"
+IUSE=""
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+RDEPEND=""
+
+python_test() {
+	if [[ ${EPYTHON} == python2* || ${EPYTHON} == pypy ]]; then
```
Comment (mgorny) [inline dev-python/typing/typing-3.7.4.1.ebuild]: There's `python_is_python3`.

File: dev-python/typing/typing-3.7.4.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+		cd "${S}"/python2 || die
+	else
+		cd "${S}"/src || die
+	fi
+
+	"${PYTHON}" test_typing.py || die "tests failed under ${EPYTHON}"
```
Comment (mgorny) [inline dev-python/typing/typing-3.7.4.1.ebuild]: EPYTHON

File: dev-python/typing/typing-3.7.4.1.ebuild (hunk 2 above)
Comment (TheChymera) [reply dev-python/typing/typing-3.7.4.1.ebuild]: yes

File: dev-python/typing/typing-3.7.4.1.ebuild (hunk 1 above)
Comment (TheChymera) [reply dev-python/typing/typing-3.7.4.1.ebuild]: done

File: dev-python/typing/typing-3.7.4.1.ebuild (hunk 3 above)
Comment (TheChymera) [reply dev-python/typing/typing-3.7.4.1.ebuild]: done

File: dev-python/typing/typing-3.7.4.1.ebuild (hunk 4 above)
Comment (TheChymera) [reply dev-python/typing/typing-3.7.4.1.ebuild]: done

**Comment by juippis** [comment]
Looks like this has been bumped, 
2877b97936b42347545c603c3b6929f3c9dcf420

**Pull request closed without merging**